components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -36,10 +37,18 @@ - (void)bottomDrawerWillChangeState:(MDCBottomDrawerPresentationController *)pre
   _delegateWasCalled = YES;
 }
 
+- (void)bottomDrawerTopTransitionRatio:
```

Comment:
_Tip:_ This isn't used in the test I don't think it's necessary

Reply:
this is needed as it is a non optional method in the protocol so must be implemented